fast64_internal/oot/__init__.py

```diff
@@ -32,13 +32,14 @@ def draw(self, context):
 
         prop_split(col, context.scene, "ootDecompPath", "Decomp Path")
         col.prop(context.scene.fast64.oot, "hackerFeaturesEnabled")
-
+        col.prop(context.scene.fast64.oot, "shipOfHarkinianCompatible")
```

Comment:
I'm confused why you are calling it this considering that as far as I can tell this feature does not merely allow "compatibility" with the PC port, but rather it changes the output such that it is no longer compatible with N64. Unless there is something I am missing, this variable should be changed to be more reflective of what it does (i.e. describing this as PC port *mode*, rather than "compatibility."